app.py

```diff
@@ -1,10 +1,10 @@
 from flask import Flask, render_template, request, redirect, url_for
 import trello as trello
 import view_model as view_model
+import dotenv
```

Comment:
We can now remove this import since we're not using the relevant bit of code any more